assets/app/views/browse/route.html

```diff
@@ -0,0 +1,74 @@
+<div ng-controller="RouteController" class="content">
+  <project-page>
+    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+    <alerts alerts="alerts"></alerts>
+    <div ng-if="route">
+      <div class="col-md-12">
+        <div class="tile">
+          <dl class="dl-horizontal left">
+            <dt>Name:</dt>
+            <dd>{{route.metadata.name}}</dd>
+            <dt>Labels:</dt>
+            <dd>
+              <span ng-if="!route.metadata.labels">none</span>
```

Comment:
Is `route.metadata.labels` an object? `(route.metadata.labels | hashSize) === 0`?